google/cloud/functions/http_response.h

```diff
@@ -41,6 +41,80 @@ class HttpResponse {
 
   HttpResponse();
 
+  /**
+   * @name Common HTTP status codes.
+   */
+  //@{
+  // Start of Information responses (1XX)
+  inline static auto constexpr kContinue = 100;
+  inline static auto constexpr kSwitchingProtocol = 101;
+  inline static auto constexpr kProcessing = 102;
+  inline static auto constexpr kEarlyHints = 103;
+  // Start of Successful responses (2XX)
+  inline static auto constexpr kOkay = 200;
+  inline static auto constexpr kCreated = 201;
+  inline static auto constexpr kAccepted = 202;
+  inline static auto constexpr kNonAuthoritativeInformation = 203;
+  inline static auto constexpr kNoContent = 204;
+  inline static auto constexpr kResetContent = 205;
+  inline static auto constexpr kPartialContent = 206;
+  inline static auto constexpr kMultiStatus = 207;
+  inline static auto constexpr kAlreadyReported = 208;
+  inline static auto constexpr kIMUsed = 226;
+  // Start of Redirection messages (3XX)
+  inline static auto constexpr kMultipleChoice = 300;
+  inline static auto constexpr kMovedPermanently = 301;
+  inline static auto constexpr kFound = 302;
+  inline static auto constexpr kSeeOther = 303;
+  inline static auto constexpr kNotModified = 304;
+  inline static auto constexpr kUseProxy = 305;
+  // inline static auto constexpr kReserved = 306;
+  inline static auto constexpr kTemporaryRedirect = 307;
+  inline static auto constexpr kPermanentRedirect = 308;
+  // Start of Client Error Responses (4XX)
```

Comment:
Nit: "Responses" was in lowercase above.

Reply:
Fixed. Also removed the `Start of ` seemed redundant on a second look.